charts/sda-svc/templates/s3-inbox-ingress.yaml

```
@@ -44,6 +44,13 @@ spec:
             name: {{ template "sda.fullname" . }}-inbox
             port:
               number: {{ ternary 443 80 .Values.global.tls.enabled }}
+      - pathType: Prefix
```

```suggestion
      - pathType: Exact
```

Since we don't have implemented any sub paths here we can set it to exact.